src/tests/validation.test.ts

```diff
@@ -424,6 +425,20 @@ suite('ASL context-aware validation', () => {
                 filterMessages: [MESSAGES.UNREACHABLE_STATE]
             })
         })
+
+        test('Shows diagnostics on invalid next property of a nested state withing DistributedMap', async () => {
+            await testValidations({
+                json: documentDistributedMapInvalidNextInNestedState,
+                diagnostics: [
+                    {
+                        message: MESSAGES.INVALID_NEXT,
+                        start: [16, 30],
+                        end: [16, 53],
+                    },
+                ],
+                filterMessages: [MESSAGES.NO_TERMINAL_STATE]
```

Comment:
Shouldn't we expect `[MESSAGES.UNREACHABLE_STATE]` instead of `[MESSAGES.NO_TERMINAL_STATE]`?

Reply:
This is actually for ignoring errors but I actually realized I shouldn't have ignored it. I updated the pr.